core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/testSets/DsSalaries.kt

```diff
@@ -0,0 +1,31 @@
+package org.jetbrains.kotlinx.dataframe.testSets
+
+import org.jetbrains.kotlinx.dataframe.annotations.ColumnName
+import org.jetbrains.kotlinx.dataframe.annotations.DataSchema
+
+@Suppress("unused")
+@DataSchema
+interface DsSalaries {
```

Comment:
We need to add somewhere the link to the initial source of dataset (for example here, on that data schema), the origin, the license, the possible transformation which we did 

It's a gentleman set of actions for copying of external dataset to your codebase. If we missed it earlier, let's start from that

Reply:
I added a link here. It was downloaded as is, without modifications and is public domain. So, do we need to add anything else here?